x-pack/plugins/secops/public/components/arrows/index.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiIcon } from '@elastic/eui';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+/** Renders the body (non-pointy part) of an arrow */
+export const ArrowBody = styled.span<{ height: number }>`
+  background-color: ${props => props.theme.eui.euiColorLightShade};
+  height: ${({ height }) => `${height}px`};
+  width: 25px;
+`;
+
+export type ArrowDirection = 'arrowLeft' | 'arrowRight';
+
+/** Renders the head of an arrow */
+export const ArrowHead = pure<{
+  direction: ArrowDirection;
+}>(({ direction }) => (
+  <EuiIcon color="subdued" data-test-subj="arrow-icon" size="s" type={direction} />
+));
```

Comment:
No changes for this PR but pointing out we have two different styles within the code base over what we are doing with the `index.tsx` files. In some cases we are using them as pure or near pure "barrels":
https://basarat.gitbooks.io/typescript/docs/tips/barrel.html

Where they contain almost all `export * from "./helpers"` and then we have other index.tsx files where we are doing mixes.

No changes required, but if you were subscribed to the barrel faith you would have all of this functionality within your helper file and then this `index.tsx` would just be doing a:

```ts
export * from './helpers'
```

Within other files you would import from the folder and not the individual components. For example in the file:

x-pack/plugins/secops/public/components/source_destination/source_destination_arrows.tsx

instead of two lines of imports:

```ts
import { ArrowBody, ArrowHead } from '../arrows';
import {
  DEFAULT_ARROW_HEIGHT,
  getArrowHeightFromPercent,
  getPercent,
  hasOneValue,
} from '../arrows/helpers';
```

you would have one like so:

```ts
import {
  ArrowBody,
  ArrowHead
  DEFAULT_ARROW_HEIGHT,
  getArrowHeightFromPercent,
  getPercent,
  hasOneValue,
} from '../arrows';
```

Again, optional nit and just pointing out the pattern.

Reply:
That's a fair point, I won't make changes in this pr, but it would be ideal to have uniformity across files here.